src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 star">
```

Comment:
the class name should be stars, not a star

```suggestion
    <div class="stars--0 stars">
```